lib/simplecov-html.rb

```diff
@@ -29,6 +30,20 @@ def output_message(result)
         "Coverage report generated for #{result.command_name} to #{output_path}. #{result.covered_lines} / #{result.total_lines} LOC (#{result.covered_percent.round(2)}%) covered."
       end
 
+      # Check if branchable results supported or not
+      # Try used here to escape exceptions
+      def branchable_result?
+        defined?(SimpleCov::branchable_report) ? SimpleCov::branchable_report : false
```

Comment:
It's not clear to me what `SimpleCov::branchable_report` refers to? Should it be `SimpleCov.branchable_report`?